bootstrap.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set +x
+
+# Ask the user for the installation directory
+read -p "Enter the installation directory (e.g., '.' or 'docs'): " install_directory
+
+# Clone the starter pack repository to a temporary directory
+echo "Cloning the starter pack repository..."
+git clone --depth 1 git@github.com:canonical/sphinx-docs-starter-pack temp-starter-pack
```

Comment:
Maybe a good excuse to insist that SP users configure SSH. Will break for those using token authentication I think.

Reply:
Nope, it works just fine for me using HTTPS.
Please don't make me switch my complete setup for no reason. ;)

Reply:
Good point. I've modified the clone to use HTTPS.